src/Compilers/Core/Portable/Operations/OperationExtensions.cs

```diff
@@ -14,6 +14,14 @@ namespace Microsoft.CodeAnalysis.Operations
 {
     public static partial class OperationExtensions
     {
+        /// <summary>
+        /// Helper function to simplify the access to the function pointer signature of an FunctionPointerInvocationOperation
+        /// </summary>
+        public static IMethodSymbol GetFunctionPointerSignature(this IFunctionPointerInvocationOperation functionPointer)
```

Comment:
We should add some tests for this extension. Also consider adding it to the `TestOperationVisitor` as well.